src/c++/vdo/base/slab-depot.h

```diff
@@ -45,6 +45,9 @@
 enum {
 	/* The number of vios in the vio pool is proportional to the throughput of the VDO. */
 	BLOCK_ALLOCATOR_VIO_POOL_SIZE = 128,
+
+	/* The number of vios in the vio pool used for loading reference count data. */
+	BLOCK_ALLOCATOR_REFCOUNT_VIO_POOL_SIZE = 16,
```

Comment:
I thouched on this obliquely, but is this a good size? The largest slab will currently load with only 9 vios I think, meaning we'll never use the full pool.